src/app/routes/layout.tsx

```diff
@@ -1,22 +1,69 @@
 import EventFulLogo from '@/shared/ui/eventFulLogo';
 import { Outlet } from 'react-router-dom';
 import Sidebar from './sidebar';
+import { Button2 } from '@/shared/ui/typography';
+import LoginForm from '@/features/auth/login/ui/LoginForm';
+import { useState } from 'react';
+import { useLogout } from '@/features/auth/login/model/queries';
 
 interface LayoutProps {
   isSidebar: boolean;
 }
+
 export default function Layout({ isSidebar }: LayoutProps) {
+  const [isLoginOpen, setIsLoginOpen] = useState(false);
+  const [isLoggedIn, setIsLoggedIn] = useState(false);
```

Comment:
상태를 선언할 때 타입도 같이 적어주면 좋을거같아요!
```suggestion
  const [isLoginOpen, setIsLoginOpen] = useState<boolean>(false);
  const [isLoggedIn, setIsLoggedIn] = useState<boolean>(false);
```

Reply:
넹 useState 쓰인 파일에 모두 타입 명시했습니다~!